apps/monitoring/ithc/base/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+  - prometheus-values.yaml
```

Comment:
Filename seems to be monitoring-values.yaml in this case

Reply:
good shout, realised its wrong in perftest too. Aligned them